app/views/welcome/index.html.erb

```diff
@@ -1 +1,37 @@
+<header class="masthead">
+  <div class="container h-100">
+    <div class="row h-100 align-items-center">
+      <div class="col-12 text-center">
+        <h1 class="font-weight-light">ObituaryHub</h1>
+        <!-- insert logo image here instead of header -->
+        <br>
+        <p class="lead">a place to celebrate life<br>and honor death</p>
```

Comment:
yo, we got a new slogan! What do you all think?

Reply:
I love it! 
We can always change it to 'We're all gonna die' later if we want